Update index.html

ausHaus opened the pull request.

Apollon77 commented:
Hi @ausHaus 

can you please add infos in the pull requests what you changed? In each single pull rewquest or in this one for all the 6 from today ... because else it will be hard to determine the purpose of your changes and the correctness.
Thank you!

ausHaus commented:
Fixes some cosmetic + latest firmware (3.53).
1. Removed support DHT11 sensors
2. Added hysteresis function (ADC ports and DS18B20 sensors)
3. Deleted team "tget" (request for internal temperature sensor value) is not yet done
4. Added the ability to "Smooth" to control smooth speed PWM control.
